sdk/cmake/host-tools.cmake

```diff
@@ -15,10 +15,41 @@ function(setup_host_tools)
         set(HOST_EXE_SUFFIX ".exe")
     endif()
 
+    if(CMAKE_SYSTEM_PROCESSOR EQUAL CMAKE_HOST_SYSTEM_PROCESSOR)
+        set(HOST_TOOLS_CMAKE_COMMAND "${CMAKE_COMMAND}")
+        message("Not cross-compiling, no special host-tools cmake command")
+    elseif(MSVC)
+        message("Compiling on ${CMAKE_HOST_SYSTEM_PROCESSOR} for ${CMAKE_SYSTEM_PROCESSOR} (MSVC)")
```

Comment:
'Compiling on AMD64 for i686 (MSVC)', on the 4 "MSVC" builds:
Value mismatch? Especially for 'build-msvc-amd64', which should be in the previous case, should it not?

Reply:
We dont set CMAKE_SYSTEM_PROCESSOR in toolchain-msvc.cmake. Maybe it's time :-)

Reply:
yeah...
next up: use `ARCH`